numba/unicode.py

```diff
@@ -531,6 +557,96 @@ def find_impl(a, b):
         return find_impl
 
 
+@overload_method(types.UnicodeType, 'rfind')
+def unicode_rfind(src, sub, start=None, end=None):
+
+    if not (start is None or isinstance(start, (types.Omitted, types.Integer, types.Optional))):
```

Comment:
This probably needs to look something like https://github.com/numba/numba/pull/4204#discussion_r306189442